src/org/zaproxy/zap/extension/wappalyzer/WappalyzerPassiveScanner.java

```diff
@@ -165,20 +163,9 @@ public void setLevel(AlertThreshold level) {
 		// Does not apply.
 	}
 
-	// @Override
+	@Override
 	public boolean appliesToHistoryType(int historyType) {
-		// TODO replace with core code once available
-		// return PluginPassiveScanner.getDefaultHistoryTypes().contains(historyType);
-
-		switch (historyType) {
-		case HistoryReference.TYPE_PROXIED:
-		case HistoryReference.TYPE_ZAP_USER:
-		case HistoryReference.TYPE_SPIDER:
-		case HistoryReference.TYPE_SPIDER_AJAX:
-			return true;
-		default:
-			return false;
-		}
+		 return PluginPassiveScanner.getDefaultHistoryTypes().contains(historyType);

```

Comment:
- The minimum ZAP version ("not-before-version" in ZapAddOn.xml file) should be updated accordingly;
 - There's an extra white space before "return" that could be removed.